packages/webdriver-utils/src/metadata/desktopMetaData.js

```diff
@@ -49,7 +49,7 @@ export default class DesktopMetaData {
 
   async screenResolution() {
     return await Cache.withCache(Cache.resolution, this.driver.sessionId, async () => {
-      const data = await this.driver.executeScript({ script: 'return [(window.screen.width * window.devicePixelRatio).toString(), (window.screen.height * window.devicePixelRatio).toString()];', args: [] });
+      const data = await this.driver.executeScript({ script: 'return [parseInt(window.screen.width * window.devicePixelRatio).toString(), parseInt(window.screen.height * window.devicePixelRatio).toString()];', args: [] });
```

Comment:
Why did we add `parseInt` here ?

Reply:
resolutions should come as proper integer not as decimals. since this was desktop it was always coming as integer as dpr values are 1,2 however added it for consistent behaviour with respect to mobile.

Reply:
Got it.
In that case, does `parseInt` take ceil or floor ?

Reply:
`parseInt` takes floor values